### ISSUE TITLE
Pressing debug icons on iPhone 11 app will crash

### ISSUE THREAD

**Issue by jerryhamby** [flutter/flutter#66078]
Just updated to iOS 14 and all my Flutter apps on the phone show the icon,
but when pressing the icon the app tries to start, then it just vanishes(crashes).	

If I debug from Flutter to the device the app appears fine as long as I'm tethered.
If I disconnect the device from my Mac, the icon has the above problem.

Funny, "Stuck on white screen when debugging iOS (#63800)" seems to be working now.
This must be a Flutter problem, because it also happens when I export to my device from Xcode.
(tested a couple of times)

This crashing happens with all my previously created Flutter apps. 

iOS 14 
iPhone 11 
Flutter 1.20.4 (iMac 10.15.6)

**Comment by Imgkl** [comment]
Hello @jerryhamby,
This is a known issue. The support for the iOS 14 is on dev channel for now. If you want to switch from stable to dev, run `flutter channel dev`. FYI, The code might not be stable, yet.

**Comment by TahaTesser** [comment]
Hi @jerryhamby 
This is a duplicate of https://github.com/flutter/flutter/issues/60657#issuecomment-694319099

![ezgif com-resize (2)](https://user-images.githubusercontent.com/48603081/93466098-58ea6680-f909-11ea-8cd6-8ebbc6cbcfe3.gif)


<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel stable, 1.20.4, on Mac OS X 10.15.6 19G2021, locale en-GB)
    • Flutter version 1.20.4 at /Users/tahatesser/Code/flutter_stable
    • Framework revision fba99f6cf9 (3 days ago), 2020-09-14 15:32:52 -0700
    • Engine revision d1bc06f032
    • Dart version 2.9.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.0, Build version 12A7209
    • CocoaPods version 1.9.3

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 49.0.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.49.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.14.1

 
[✓] Connected device (2 available)            
    • Taha’s iPhone (mobile) • 00008020-001059882212002E            • ios • iOS 14.0
    • iPhone 11 (mobile)     • FF9F475E-22A5-4E7F-904F-FD5CE8D0E905 • ios •
      com.apple.CoreSimulator.SimRuntime.iOS-14-0 (simulator)

• No issues found!
```

</details> 


Please check this https://github.com/flutter/flutter/issues/60657#issuecomment-694410188
Thank you

**Comment by jerryhamby** [comment]
I've just upgraded my Mac to 10.15.7 and this issue is worst now.

I'm running my app (from VSC) to my physical device (iPhone 11), this use to work as long as I was connected and stayed connected using a wire to my device. Now it goes to a "white screen" and displays the error: 
Exited (sigterm)
Could not update files on device: HttpException: , uri = http://localhost:50029

2- If I try to use Xcode to load my app to the device, I now get another error: 
Undefined symbol: _SDImageCoderEncodeCompressionQuality

Bottom line, I'm locked out from testing with my physical device, until someone on the Flutter team decides to fix things. 
I don't think I'm the only one.  I'm in the final stages of testing and this is a SUPER problem in my efforts...

iMac: 10.15.7 
Flutter: 1.20.4
VSC: 1.49.2
iPhone 11: 14.0

**Comment by jerryhamby** [comment]
A note about further testing:

I decided to test on my iPhone 7 device (with older OS) (on VSC) and things worked fine with one big exception.
It still gets the "white screen" for 3-4 minutes before it will load. 
Can't test on my iPhone 11 device, but I can now with the iPhone 7.

iMac: 10.15.7
Flutter: 1.20.4
VSC: 1.49.2
iPhone 7: 13.5.1

**Comment by TahaTesser** [comment]
Hi @jerryhamby 
The white screen is this https://github.com/flutter/flutter/issues/63800
